internal/venice-common/src/main/java/com/linkedin/venice/kafka/validation/Segment.java

```diff
@@ -39,6 +42,16 @@
  */
 @NotThreadsafe
 public class Segment {
+  /**
+   * This cache is to reduce the size on heap of debug info, which is very repetitive in nature. Using this cache, each
```

Comment:
Do you have a few examples handy to show how the debugging info look like?
I checked a few places where `debugInfo` is written and most of places are either an empty map or null.

Reply:
Sure, you can see the handful of properties we include in this map in the `Utils::getDebugInfo` function. It includes the pid/path/hostname/user of the producing process, the Java version and (if available on the classpath...) the Venice jar version.

Reply:
I included all debug info properties in the unit test.